src/main/java/org/fundacionjala/virtualassistant/repository/WhisperClient.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.virtualassistant.repository;
+
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+
+public class WhisperClient implements ASRClient {
+    private static final String URL = "http://localhost:5000";
```

Comment:
this url should be defined in application.properties, so it could be easily changed